pkg/monitor/cluster/jobconditions.go

```diff
@@ -0,0 +1,54 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/Azure/ARO-RP/pkg/util/namespace"
+)
+
+var jobConditionsExpected = map[batchv1.JobConditionType]v1.ConditionStatus{
+	batchv1.JobComplete: v1.ConditionTrue,
+	batchv1.JobFailed:   v1.ConditionFalse,
+}
+
+func (mon *Monitor) emitJobConditions(ctx context.Context) error {
+
+	jobs, err := mon.cli.BatchV1().Jobs("").List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	for _, job := range jobs.Items {
+		if !namespace.IsOpenShift(job.Namespace) {
```

Comment:
This function returns true if the namespace is ==  "default".  Do we want to emit metrics for this namespace?

Reply:
Since  we already do that for pod and container condition, replicasets and other condition/statuses metrics, I think we should do the same for job condition in order to be consistent accross the metrics.